.copr/Makefile

```diff
@@ -0,0 +1,17 @@
+srpm:
+	# Setup development environment
+	echo "Installing base development environment"
+	dnf install -y gdb make autoconf rpm-build gettext-devel automake libtool docker python3-paramiko python3-pyyaml
```

Comment:
Why does COPR auto-build need paramiko, docker, and pyyaml?

Reply:
You also need ``dnf-plugins-core`` for ``dnf builddep`` command.

Reply:
As per @abbra 's comment, I tried to reuse the same template as `ipatests/azure/templates/prepare-build.yml` to keep things uniform.

Reply:
Removed unnecessary packages in my latest commit